certified-connectors/MongoDB/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1421 @@
+              "properties": {
+                "deletedCount": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "deletedcount"
+                }
+              }
+            },
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              },
+              "Accept": {
+                "description": "Accept",
+                "type": "string"
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request",
+            "schema": {},
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              },
+              "Accept": {
+                "description": "Accept",
+                "type": "string"
+              }
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "schema": {},
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              },
+              "Accept": {
+                "description": "Accept",
+                "type": "string"
+              }
+            }
+          }
+        }
+      }
+    },
+    "/action/insertMany": {
+      "post": {
+        "summary": "Insert Multiple Documents",
+        "description": "You can add more than one document at a time by using the insertmany endpoint. In that case, you must use the documents property in the body of the request to specify the array of documents you want to insert into the collection.",
+        "operationId": "InsertMultipleDocuments",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "x-ms-summary": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/{{CONTENT_TYPE}}",
+            "description": "Content-Type"
+          },
+          {
+            "name": "Access-Control-Request-Headers",
+            "x-ms-summary": "Access-Control-Request-Headers",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "*",
+            "description": "Access-Control-Request-Headers"
+          },
+          {
+            "name": "api-key",
+            "x-ms-summary": "api-key",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "{{API_KEY}}",
+            "description": "api-key"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "dataSource": {
+                  "type": "string",
+                  "description": "datasource",
```

Comment:
@DianaMDB make sure descriptions are grammatically correct and free of typographical errors. You can refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).